docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -180,6 +188,16 @@ fields:
       - docassemble.ALMassachusetts>=0.0.7
       - docassemble.MassAccess
 ---
+objects: 
+  - tmp_allowed_courts: DAEmpty 
+  - tmp_categories: DAEmpty 
+---
+code: |    
+  if not interview.court_related:   
+    interview.form_type = 'non_court'          
+    interview.allowed_courts = tmp_allowed_courts #checkboxes               
+    interview.categories = tmp_categories  #checkboxes
```

Comment:
```suggestion
    interview.allowed_courts = DAEmpty()              
    interview.categories = DAEmpty()
```
Seems to work for me. Much easier to understand, and doesn't need the tmp objects block. I don't think that making the tmp objects persistent like that does much.